mocks/controllers/BasicAuthenticationController.go

```diff
@@ -9,16 +9,20 @@ import (
 	mock "github.com/stretchr/testify/mock"
 )
 
-// BasicAuthentificationController is an autogenerated mock type for the BasicAuthentificationController type
-type BasicAuthentificationController struct {
+// BasicAuthenticationController is an autogenerated mock type for the BasicAuthenticationController type
+type BasicAuthenticationController struct {
 	mock.Mock
 }
 
 // BasicLoginHandler provides a mock function with given fields: _a0, _a1
-func (_m *BasicAuthentificationController) BasicLoginHandler(_a0 http.ResponseWriter, _a1 *http.Request) (interface{}, httperrors.HTTPError) {
+func (_m *BasicAuthenticationController) BasicLoginHandler(_a0 http.ResponseWriter, _a1 *http.Request) (interface{}, httperrors.HTTPError) {
 	ret := _m.Called(_a0, _a1)
 
 	var r0 interface{}
+	var r1 httperrors.HTTPError
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.